lib/Runtime/Base/FunctionBody.cpp

```diff
@@ -9256,7 +9253,6 @@ namespace Js
                 {
                     newEntryPoint = simpleJitEntryPointInfo;
                     functionBody->SetDefaultFunctionEntryPointInfo(simpleJitEntryPointInfo, newEntryPoint->GetNativeEntrypoint());
-                    functionBody->SetExecutionMode(ExecutionMode::SimpleJit);
```

Comment:
Are we missing an udpate to the execution state here?

Reply:
nope--it was moved into FunctionExecutionStateMachine::ResetSimpleJitLimit (the next function call)